resources/build/help-keyman-com.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+
+#
+# Prevents 'clear' on exit of mingw64 bash shell
+#
+SHLVL=0
+
+set -e
+set -u
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+#
+# Allows us to check for existence of subfolders in help/
+#
+shopt -s nullglob
+
+#
+# In this script we update the git repo and copy the Keyman for [Platform]
+# documentation over. This should be run after a full build.
+#
+
+#
+# These are passed via environment:
+#
+# HELP_KEYMAN_COM = the home of the help.keyman.com repository
+#
+# That repo must have push to origin configured and logged in.
+# Note: GitHub API only available via HTTPS, and not SSH
+# https://github.com/github/hub/issues/1644#issuecomment-359002547
+#
+
+if [ -z ${HELP_KEYMAN_COM+x} ]; then
+  >&2 echo "Not uploading documentation: must set HELP_KEYMAN_COM in environment."
+  exit 1
+fi
+
+if [ ! -d "$HELP_KEYMAN_COM/products/" ]; then
+  >&2 echo "HELP_KEYMAN_COM path ($HELP_KEYMAN_COM) does not appear to be valid."
+  exit 1
+fi
+
+function display_usage {
+  echo "Usage: $0 [platform]"
+  echo "       $0 -help"
+  echo
+  echo "  platform should be one of: android, ios, linux, mac, windows."
+  echo "  -help               displays this screen and exits"
+  exit 1
+}
+
+#
+# Define terminal colours
+#
+
+if [ -t 2 ]; then
+  t_red=$'\e[1;31m'
+  t_grn=$'\e[1;32m'
+  t_yel=$'\e[1;33m'
+  t_blu=$'\e[1;34m'
+  t_mag=$'\e[1;35m'
+  t_cyn=$'\e[1;36m'
+  t_end=$'\e[0m'
+fi
```

Comment:
had to add this block for the warning in l.114

I guess we don't have a standard util.sh for terminal colors